src/main/java/org/folio/consortia/service/HttpRequestServiceImpl.java

```diff
@@ -31,6 +33,7 @@ public class HttpRequestServiceImpl implements HttpRequestService {
   @Override
   public PublicationHttpResponse performRequest(String url, HttpMethod httpMethod, Object payload) {
     var headers = convertHeadersToMultiMap(folioExecutionContext.getOkapiHeaders());
+    headers.setAccept(Collections.singletonList(MediaType.TEXT_PLAIN));
```

Comment:
I think this header can break other requests

Reply:
i have tested apis manually , all working fine.

Reply:
we use now */* accept header, so this comment is not relevant any more